infra/client.go

```diff
@@ -12,6 +12,13 @@ import (
 func CreateGRPCClient(certs [][]byte) (*comm.GRPCClient, error) {
 	config := comm.ClientConfig{}
 	config.Timeout = 5 * time.Second
+	config.KaOpts = &comm.KeepaliveOptions{
```

Comment:
@guoger , after add those changes to grpc clients. I am able to use 
  "num_of_conn": 1,
  "client_per_conn": 1
with 100 blocks at my local laptop.

Guessing, the grpc client performance impacts total performance.